iac-extensions/docker/src/main/java/org/sonar/iac/docker/checks/utils/FlagNoSpaceArgumentPredicate.java

```diff
@@ -30,15 +30,18 @@
  */
 public class FlagNoSpaceArgumentPredicate implements Predicate<ArgumentResolution> {
   private final String flag;
+  private final boolean ignoreQuotes;
 
-  public FlagNoSpaceArgumentPredicate(String flag) {
+  public FlagNoSpaceArgumentPredicate(String flag, boolean ignoreQuotes) {
```

Comment:
I added it for consistency, because all other predicates are wrapped into `*IgnoreQuotes`. But I agree, as it is always set to true, I'll remove the flag